### ISSUE TITLE
Error caused by require TransitionGroup module directly 

### ISSUE THREAD

**Issue by yvesyao** [google-fabric/velocity-react#250]
I encountered an error today:
```
react.development.js:369 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.

Check the render method of `VelocityTransitionGroup`.
    in VelocityTransitionGroup (created by TreeNode)
```
Then I debugged into the error part and found that VelocityTransitionGroup created a child of Object:
![image](https://user-images.githubusercontent.com/7723729/54920943-825a4a00-4f3f-11e9-9b28-f6872e8bee1c.png)

The following codes shows that this error is caused by requiring TransitionGroup module directly rather than require the default part of the module.
https://github.com/google-fabric/velocity-react/blob/v1.4.1/src/velocity-transition-group.js#L56
https://github.com/reactjs/react-transition-group/blob/v2.7.0/src/TransitionGroup.js#L169

**Comment by jmauzy** [comment]
This appears to be related to a breaking minor bump here:
https://github.com/reactjs/react-transition-group/issues/475

**Comment by joelhoelting** [comment]
I know that an update is coming soon but have you guys found any fixes for this?

**Comment by log0s** [comment]
@joelhoelting For now you can just explicitly require version 2.6.1 of `react-transition-group`

**Comment by yvesyao** [comment]
This issue has been resolved in version 2.7.1:
[reactjs/react-transition-group#475](https://github.com/reactjs/react-transition-group/issues/475)
